Issue with Home Assistant Zabbix_Agent2 on fresh install of Home Assistant OS 12.3

Following a fresh install of Home Assistant OS on a Proxmox VM, the zabbix_agent2 plugin will not start.

Logs have this repeating:

```
goroutine 1 [running]:
golang.zabbix.com/agent2/plugins/external.(*Plugin).ExecutePlugin(0xc000125cc0)
	golang.zabbix.com/agent2/plugins/external/plugin.go:67 +0x4d9
main.initExternalPlugin({0xc00012ac00, 0xa}, 0xc000125cc0, 0x55cf8bfa0300)
	golang.zabbix.com/agent2/cmd/zabbix_agent2/external.go:95 +0x5d
main.initExternalPlugins(0x55cf8bfa0300)
	golang.zabbix.com/agent2/cmd/zabbix_agent2/external.go:79 +0x432
main.main()
	golang.zabbix.com/agent2/cmd/zabbix_agent2/zabbix_agent2.go:375 +0x58a
panic: failed to start plugin /usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql, fork/exec /usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql: no such file or directory
```

![image](https://github.com/pschmitt/home-assistant-addons/assets/1345467/b14e6170-0dce-4439-aec3-edbf08f4ad72)

All other addons seem to be ok, which is good, I can help provide anything needed.

I'm also getting the same:

```main.main()
	golang.zabbix.com/agent2/cmd/zabbix_agent2/zabbix_agent2.go:375 +0x58a
panic: failed to start plugin /usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql, fork/exec /usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql: no such file or directory

goroutine 1 [running]:
golang.zabbix.com/agent2/plugins/external.(*Plugin).ExecutePlugin(0xc0000b7ae0)
	golang.zabbix.com/agent2/plugins/external/plugin.go:67 +0x4d9
main.initExternalPlugin({0xc000014c90, 0xa}, 0xc0000b7ae0, 0x55f5a38b9300)
	golang.zabbix.com/agent2/cmd/zabbix_agent2/external.go:95 +0x5d
main.initExternalPlugins(0x55f5a38b9300)
	golang.zabbix.com/agent2/cmd/zabbix_agent2/external.go:79 +0x432
main.main()
	golang.zabbix.com/agent2/cmd/zabbix_agent2/zabbix_agent2.go:375 +0x58a
panic: failed to start plugin /usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql, fork/exec /usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql: no such file or directory

goroutine 1 [running]:
golang.zabbix.com/agent2/plugins/external.(*Plugin).ExecutePlugin(0xc0000b7b80)
	golang.zabbix.com/agent2/plugins/external/plugin.go:67 +0x4d9
main.initExternalPlugin({0xc000014c90, 0xa}, 0xc0000b7b80, 0x563868d12300)
	golang.zabbix.com/agent2/cmd/zabbix_agent2/external.go:95 +0x5d
main.initExternalPlugins(0x563868d12300)
	golang.zabbix.com/agent2/cmd/zabbix_agent2/external.go:79 +0x432
main.main()
	golang.zabbix.com/agent2/cmd/zabbix_agent2/zabbix_agent2.go:375 +0x58a
panic: failed to start plugin /usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql, fork/exec /usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql: no such file or directory
```

Same for me

```goroutine 1 [running]:
golang.zabbix.com/agent2/plugins/external.(*Plugin).ExecutePlugin(0xc0000a9ae0)
	golang.zabbix.com/agent2/plugins/external/plugin.go:67 +0x4d9
main.initExternalPlugin({0xc000014c80, 0xa}, 0xc0000a9ae0, 0x55a32a6d6300)
	golang.zabbix.com/agent2/cmd/zabbix_agent2/external.go:95 +0x5d
main.initExternalPlugins(0x55a32a6d6300)
	golang.zabbix.com/agent2/cmd/zabbix_agent2/external.go:79 +0x432
main.main()
	golang.zabbix.com/agent2/cmd/zabbix_agent2/zabbix_agent2.go:375 +0x58a
panic: failed to start plugin /usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql, fork/exec /usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql: no such file or directory

goroutine 1 [running]:
golang.zabbix.com/agent2/plugins/external.(*Plugin).ExecutePlugin(0xc00011fae0)
	golang.zabbix.com/agent2/plugins/external/plugin.go:67 +0x4d9
main.initExternalPlugin({0xc000124c00, 0xa}, 0xc00011fae0, 0x55cc1b454300)
	golang.zabbix.com/agent2/cmd/zabbix_agent2/external.go:95 +0x5d
main.initExternalPlugins(0x55cc1b454300)
	golang.zabbix.com/agent2/cmd/zabbix_agent2/external.go:79 +0x432
main.main()
	golang.zabbix.com/agent2/cmd/zabbix_agent2/zabbix_agent2.go:375 +0x58a```

--- a/zabbix-agent2/CHANGELOG.md
+++ b/zabbix-agent2/CHANGELOG.md
@@ -1,3 +1,9 @@
+# [1.5.1] - 2024-09-03
+
+## Fixed
+
+- Adjust configuration to upstream plugin executable name changes
+
 # [1.5] - 2023-07-10
 
 ## Added

--- a/zabbix-agent2/Dockerfile
+++ b/zabbix-agent2/Dockerfile
@@ -18,7 +18,7 @@ RUN apk add --no-cache jq zabbix-agent2 && \
 
 # Copy zabbix-agent2 plugins
 COPY --from=builder ["/usr/sbin/zabbix-agent2-plugin", "/usr/sbin/zabbix-agent2-plugin"]
-RUN echo -e "\nPlugins.PostgreSQL.System.Path=/usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-mongodb\nPlugins.PostgreSQL.System.Path=/usr/sbin/zabbix-agent2-plugin/zabbix-agent2-plugin-postgresql\n" >> /etc/zabbix/zabbix_agent2.conf
+RUN echo -e "\nPlugins.PostgreSQL.System.Path=/usr/sbin/zabbix-agent2-plugin/mongodb\nPlugins.PostgreSQL.System.Path=/usr/sbin/zabbix-agent2-plugin/postgresql\n" >> /etc/zabbix/zabbix_agent2.conf
 
 # Copy data for add-on
 COPY run.sh /

--- a/zabbix-agent2/Dockerfile
+++ b/zabbix-agent2/Dockerfile
@@ -1,10 +1,10 @@
 ARG BUILD_ARCH
 ARG BUILD_VERSION
-ARG BUILD_BASE_IMAGE=zabbix/zabbix-agent2:alpine-latest
+ARG BUILD_BASE_IMAGE=zabbix/zabbix-agent2:alpine-6.4-latest
 
 FROM ${BUILD_BASE_IMAGE} as builder
 
-FROM alpine
+FROM alpine:3.20
 
 LABEL maintainer "Philipp Schmitt <philipp@schmitt.co>"
 

--- a/zabbix-agent2/config.json
+++ b/zabbix-agent2/config.json
@@ -1,6 +1,6 @@
 {
   "name": "zabbix-agent2",
-  "version": "1.4",
+  "version": "1.5.1",
   "slug": "zabbix-agent2",
   "description": "Zabbix agent 2",
   "url": "https://github.com/pschmitt/home-assistant-addons/tree/main/zabbix-agent2",

